### PR TITLE
Dpath pert

### PR DESCRIPTION
**Pull request by jpa198**
#### Checklist
- Have you updated the README (yes/no)?
- Have you updated the CHANGELOG (yes/no)?

### CODE REVIEW

**Comment by jpa198** [comment]
Pending e2e test

File: e2e_tests/E2ETests.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 import numpy.random as nprdm
 import pandas as pd
 import sqlite3 as sql
-from pandas.util.testing import assert_frame_equal
+from pandas.testing import assert_frame_equal
```
Comment (jasonmotha) [inline e2e_tests/E2ETests.py:18]: Hmmmm, using testing module?

Comment (jasonmotha) [reply e2e_tests/E2ETests.py:18]: What's the future plans for the function?

Comment (jpa198) [reply e2e_tests/E2ETests.py:18]: Using the util testing gave an error saying "Use the pandas.testing api version, not this one"
The version update is for the atol/rtol arguments of the function

Comment (jpa198) [reply e2e_tests/E2ETests.py:18]: The function was already being used to compare the benchmark and test im_csv files, if that's what you mean?

**Pull request merged**